src/components/turbo-mode/turbo-mode.jsx

```diff
@@ -15,7 +15,7 @@ const TurboMode = () => (
             <FormattedMessage
                 defaultMessage="Turbo Mode"
                 description="Label indicating turbo mode is active"
-                id="controls.turboMode"
+                id="gui.turboMode"
```

Comment:
What about when the string and the component name are the same? Should this be `gui.turboMode.turboMode` for consistency, or is that just overkill? Especially when there's only one string in the component.